### PR TITLE
Remove isViaExpensifyCash

### PR DESCRIPTION
**Pull request by stitesExpensify**
@marcaaron cc: @AndrewGable 
### Details
Something wonky is happening where we didn't deploy the last branch. Hopefully this will fix things

### Fixed Issues
Fire:
https://expensify.slack.com/archives/C011W8BJ9L6/p1610505357445700?thread_ts=1610502517.428200&cid=C011W8BJ9L6

### Tests
Made sure I could go through the login flow on web

### CODE REVIEW

**Review by marcaaron — APPROVED** [review]

File: src/libs/API.js [hunk 1]
```diff
@@ -237,7 +237,6 @@ function Authenticate(parameters) {
         partnerUserSecret: parameters.partnerUserSecret,
         twoFactorAuthCode: parameters.twoFactorAuthCode,
         doNotRetry: true,
-        isViaExpensifyCash: true,
```
Comment (AndrewGable) [inline src/libs/API.js:240]: I'm not sure we wanted to remove this line

Comment (stitesExpensify) [reply src/libs/API.js:240]: ah crap looks like you're right. Why didn't we change that one in api.php as well?

Comment (marcaaron) [reply src/libs/API.js:240]: Why not? I'm not seeing it...

Comment (stitesExpensify) [reply src/libs/API.js:240]: We forgot to update the command in api.php to look for the referer, so it was possible to authenticate through e.cash without a GH token. 
CP'd this fix https://github.com/Expensify/Web-Expensify/pull/30071

Comment (marcaaron) [reply src/libs/API.js:240]: Ah got it. I was looking at `master` wondering what the problem was but you've fixed it 😅

**Pull request merged**